Doc/library/typing.rst

```diff
@@ -1022,6 +1024,18 @@ These can be used as types in annotations using ``[]``, each having a unique syn
 
    .. versionadded:: 3.8
 
+.. data:: Required
+
+.. data:: NotRequired
+
+   A special typing construct that marks individual keys of a :class:`TypedDict`
+   as either required or non-required respectively.
+
+   For more information, see
```

Comment:
You've added some good docs in this file; we should link to those more prominently than the PEP, which may grow out of date since it is now a historical document.

```suggestion
   For more information, see :class:`TypedDict` and
```